crates/uv/src/commands/tool/dir.rs

```diff
@@ -3,16 +3,23 @@ use owo_colors::OwoColorize;
 
 use uv_configuration::PreviewMode;
 use uv_fs::Simplified;
-use uv_tool::InstalledTools;
+use uv_tool::{find_executable_directory, InstalledTools};
 use uv_warnings::warn_user_once;
 
 /// Show the tool directory.
-pub(crate) fn dir(preview: PreviewMode) -> anyhow::Result<()> {
+pub(crate) fn dir(bin: bool, preview: PreviewMode) -> anyhow::Result<()> {
     if preview.is_disabled() {
         warn_user_once!("`uv tool dir` is experimental and may change without warning.");
     }
-    let installed_tools =
-        InstalledTools::from_settings().context("Failed to initialize tools settings")?;
-    anstream::println!("{}", installed_tools.root().simplified_display().cyan());
+
+    if bin {
+        let executable_directory = find_executable_directory()?;
+        anstream::println!("{}", executable_directory.simplified_display().cyan());
```

Comment:
Is there a reason you opted for the full name here? I think it's standard to do

```
use anstream::{eprintln, println};
```

nearly everywhere else.

Reply:
Consistency with the rest of the file.

Reply:
Fixed: https://github.com/astral-sh/uv/pull/5187